pkgs/development/libraries/opencv/default.nix

```diff
@@ -81,6 +83,6 @@ stdenv.mkDerivation rec {
     homepage = http://opencv.org/;
     license = stdenv.lib.licenses.bsd3;
     maintainers = with stdenv.lib.maintainers; [viric flosse];
-    platforms = with stdenv.lib.platforms; linux;
+    platforms = with stdenv.lib.platforms; linux ++ darwin;
```

Comment:
I'm not sure what's going on here, but it seems like the `darwin` variable here is shadowed by the function argument for some reason.

```
error: value is a set while a list was expected, at pkgs/development/libraries/opencv/default.nix:86:44
```